config/initializers/search_backend.rb

```diff
@@ -1,5 +1 @@
-Whitehall.search_backend = if Rails.env.test?
```

Comment:
I hope this is OK?  I'm not sure how much confidence I have in the tests catching whether this will still work afterwards given the liberal use of fake things.

Reply:
Yeah, I didn't have *too* much confidence in the tests, but the e2e tests ran fine.  Smokey and traffic replay should help too.